calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx

```diff
@@ -0,0 +1,8 @@
+xx May 2023
+
+### What's new {#whats-new-3.17.0}
+
+### Known issues {#known-issues-3.17.0}
+
+* L7 log collection fails to deploy on CIS hardened clusters. As a result, some cards in the manager UI dashboard will not display any metrics.
```

Comment:
This can be removed. It is no longer an issue from CE v3.16.1.

Reply:
Removed